docs/documentation/helm.md

```diff
@@ -0,0 +1,277 @@
+
+However, users can provide other values using the `value` property:
+
+```
+dekorate.helm.name=myChart
+dekorate.helm.description=Description of my Chart
+
+# Map all the metadata name resources
+dekorate.helm.values[0].property=myModule.name
+dekorate.helm.values[0].jsonPaths=$..metadata.name
+dekorate.helm.values[0].value=this-is-another-name
+```
+
+And the `values.yaml` file will now contain:
+
+```yaml
+myModule:
+  name: this-is-another-name
+```
+
+##### Mapping multiple properties at once
+
+What about if the properties to map are at different locations, for example:
+
+```yaml
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: helm-on-kubernetes-example ## we need to map this property
+spec:
+  rules:
+    - host: my-host
+      http:
+        paths:
+          - backend:
+              service:
+                name: helm-on-kubernetes-example ## And this property
+                port:
+                  name: http
+            path: /
+            pathType: Prefix
+```
+
+For this scenario, you need to provide a comma-separated list of JSONPath expressions to be mapped to the same property `myModule.name`:
+
+```
+dekorate.helm.name=myChart
+dekorate.helm.description=Description of my Chart
+
+# Map all the metadata name resources
+dekorate.helm.values[0].property=myModule.name
+dekorate.helm.values[0].jsonPaths=$..metadata.name,$.[?(@.kind == 'Ingress')].spec.rules..http.paths..backend.service.name
+```
+
+So, Dekorate will first map the expression `$..metadata.name` and then the expression `$.[?(@.kind == 'Ingress')].spec.rules..http.paths..backend.service.name` (this expression only applies to `Ingress` resources - see more about filtering in JSONPath).
+
+#### Helm Profiles
+
+By default, all the properties are mapped to the same Helm values file `values.yaml`. However, Dekorate also supports the generation of Helm values by profiles. 
+For example, let's say we have two environments: one for testing and another one for production; each environment have a different ingress host where your Kubernetes applications will be exposed. 
+We can configure our application as:
+
+```
+dekorate.kubernetes.expose=true
+# Mapped to `values.yaml` by the preconfigured Ingress decorator
+dekorate.kubernetes.host=my-host
+
+# Helm Chart
+dekorate.helm.name=myChart
+## Overwrite the value of `dekorate.kubernetes.host` to `values-<profile-name>.yaml`:
+dekorate.helm.values[0].property=myModule.host
+dekorate.helm.values[0].jsonPaths=$.[?(@.kind == 'Ingress')].spec.rules..host
+dekorate.helm.values[0].value=my-test-host
+dekorate.helm.values[0].profile=test
+```
+
+This configuration will generate the `values.yaml` using the property `dekorate.kubernetes.host`:
+
+```yaml
+myModule:
+  host: my-host
+```
+
+But as you are now using a profile named `test` in one of your mapped properties, it will also generate a `values-test.yaml` file with the content:
+
+```yaml
+myModule:
+  host: my-test-host
+```
+
+#### Helm Usage
```

Comment:
Align this text with previous remark :up:

Reply:
What do you mean? I think it's ok as it is. The sections are:
- Getting Started
- Building
- Mapping Values
- Helm Profiles
- Helm Usage

All are at the same level.